src/plugins/intel_cpu/tests/functional/single_layer_tests/conversion.cpp

```diff
@@ -52,8 +52,17 @@ class ConvertCPULayerTest : public testing::WithParamInterface<convertLayerTestP
         std::tie(shapes, inPrc, outPrc, cpuParams) = GetParam();
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
-
-        selectedType = std::string("unknown_") + (inPrc == InferenceEngine::Precision::U8 ? "I8" : inPrc.name());
+        auto primitive = selectedType;
+        if (primitive.empty())
+            primitive = getPrimitiveType();
+        // WA: I32 precision support disabled in snippets => primitive has to be changed
```

Comment:
Could you please add a todo + ticket number (ideally also add a link in the ticket), so we won't forget to remove this WA when I32 is supported. I believe @eshoguli can give you the ticket id

Reply:
Todo and ticket number are added. However, we will have to remove this WA anyway because these tests will fail when I32 precision is enabled for snippets